tests/RobotFramework/tests/tedge/tedge_config_get.robot

```diff
@@ -57,3 +63,8 @@ Check known tedge environment settings
     [Arguments]    ${ENV_NAME}    ${KEY_NAME}    ${VALUE}
     ${output}=    Execute Command    env ${ENV_NAME}\=${VALUE} tedge config get ${KEY_NAME}
     Should Be Equal    ${output}    ${VALUE}\n
+
+Check known tedge environment settings for topics
+    [Arguments]    ${ENV_NAME}    ${KEY_NAME}    ${VALUE}
+    ${output}=    Execute Command    env ${ENV_NAME}\=${VALUE} tedge config get ${KEY_NAME}
+    Should Be Equal    ${output}    ["topic/1", "topic/2/+", "topic/3/#"]\n
```

Comment:
It is slightly strange comparing the output with a fixed expected value when the calling function provides the `${VALUE}`...so we are relying on the fact that the `${VALUE}` is the same on each call.

It would probably make sense doing one of the following:

1. Just use a fixed `${VALUE}` instead of letting the test call it (so it can always be aligned with the expected value)
2. Add a new value called `${EXPECTED_VALUE}`, which can be included in the test template (this will the user worry about the sync between the `${VALUE}` and `${EXPECTED_VALUE}`.

Reply:
Thanks. I took the approach 1) as it seems simpler.